modules/adopter-registration-impl/pom.xml

```diff
@@ -58,6 +58,14 @@
       <groupId>jakarta.ws.rs</groupId>
       <artifactId>jakarta.ws.rs-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient-osgi</artifactId>
```

Comment:
Why use an external library now that [Java has a built-in HTTP client](https://www.baeldung.com/java-9-http-client)? Does it not support HTTP forward proxies?

Reply:
I was cribbing from known working code (or at least code that no one's complaining about :smile:), which pulled this in.  I don't really care, but I also don't have an institutional proxy to test with...